admin/c2cgeoportal_admin/views/layers_wmts.py

```diff
@@ -53,7 +56,15 @@ def grid(self):
                  request_method='GET',
                  renderer='../templates/edit.jinja2')
     def view(self):
-        return super().edit()
+        _view = super().edit()
```

Comment:
this is usally named response, note that _ (protected) make do not really make sense inside a method.